prow/config.yaml

```diff
@@ -3176,6 +3176,88 @@ postsubmits:
         secret:
           secretName: service-account
 
+  kubernetes/cluster-registry:
+  - name: pull-cluster-registry-bazel-build
+    agent: kubernetes
+    context: pull-cluster-registry-bazel
+    always_run: true
+    rerun_command: "/test pull-cluster-registry-bazel-build"
+    trigger: "(?m)^/test( all| pull-cluster-registry-bazel-build),?(\\s+|$)"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/bazelbuild:v20171013-53473ec5-0.6.1
+        args:
+        - "--clean"
+        - "--git-cache=/root/.cache/git"
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--" # end bootstrap args, scenario args below
+        - "build"
+        env:
+        - name: TEST_TMPDIR
+          value: /root/.cache/bazel
+        # Bazel needs privileged mode in order to sandbox builds.
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: service
+          mountPath: /etc/service-account
+          readOnly: true
+        - name: cache-ssd
+          mountPath: /root/.cache
+        ports:
+        - containerPort: 9999
+          hostPort: 9999
+      volumes:
+      - name: service
+        secret:
+          secretName: service-account
+      - name: cache-ssd
+        hostPath:
+          path: /mnt/disks/ssd0
+  - name: pull-cluster-registrry-bazel-test
```

Comment:
do you want to make the test job to run-after-success the build job? btw, s/registrry/registry/

Reply:
you probably don't need a seperate build and test job, we use seperate jobs since testing adds some time to the job and we have a job that won't run until the build job is complete, but for this case I would combine these two jobs.

Reply:
actually I looked at https://github.com/kubernetes/cluster-registry/pull/41/files, you probably _do_ need seperate jobs :blush:, but you definitely need to add entries in config.json for each job that match and then run `bazel run //jobs:config_sort`

Reply:
Ah, OK, combining them does make sense here.